### ISSUE TITLE
Distribute MATLAB binaries with conda?

### ISSUE THREAD

**Issue by roversch** [conda-forge/casadi-feedstock#5]
Would it be possible to have the MATLAB mex files in the feedstock as well?

**Comment by pstjohn** [comment]
Yeah thats a good question. Im not immediately coming up with any good ways of doing it, since it doesn't look like there is an octave conda recipe anywhere. I'm also not clear on how the official binaries do it.

As I don't really use MATLAB I probably wouldn't be able to put in the time to get it working, but if you'd like to submit a PR I'm happy to include it!

**Comment by roversch** [comment]
> I'm also not clear on how the official binaries do it.

What do you mean with 'official'?

Also, the Octave binaries should work for MATLAB as well, right?

**Comment by pstjohn** [comment]
The casadi team maintains the downloads at casadi.org with this repo: https://github.com/casadi/binaries

Browsing the source there indicates that they may use a mix of octave and some MATLAB distributions to get the MATLAB distribution working. @jgillis would probably know more 😄 . 

This install option is pretty much just maintained by me (not affiliated with casadi) as it has a couple advantages (and several disadvantages) over their binary install option.

**Comment by traversaro** [comment]
I thought I mentioned this here, but apparently I did not. We had `casadi-matlab-bindings` packages for CasADi MATLAB binaries that build on top of the casadi conda-forge binaries in the `robotology` channel since the last three years, see:
* https://anaconda.org/robotology/casadi-matlab-bindings
* https://prefix.dev/channels/robotology/packages/casadi-matlab-bindings

It should be possible to easily install them with:

~~~
conda create -n casadi-matlab -c conda-forge -c robotology casadi-matlab-bindings
~~~

The package appends the location of the MATLAB libraries inside the location of the `MATLABPATH` environment variables (see https://it.mathworks.com/help/matlab/matlab_env/add-folders-to-matlab-search-path-at-startup.html#btpajlw), so as long as you launch MATLAB after activating the conda environment with `casadi-matlab-bindings`.

The packages are built using the CMake machinery available at https://github.com/ami-iit/casadi-matlab-bindings , and using GitHub Actions with https://github.com/matlab-actions/setup-matlab.

To see an example of their use in a real world scenario, check https://github.com/ami-iit/paper_bergonti_2022_tro_kinematics-control-morphingcovers, where `pixi` is used to easily install `casadi` matlab bindings and run the experiments of the paper.

It may be possible to build something similar directly on conda-forge, but it may require some work to understand how to install MATLAB on the Azure machines used by conda-forge CI.
